File: integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/fiks/svarinn/SvarInnPackage.java
package no.difi.meldingsutveksling.fiks.svarinn;

import no.difi.meldingsutveksling.domain.sbdh.StandardBusinessDocument;
import org.springframework.core.io.Resource;

public class SvarInnPackage {

    private final StandardBusinessDocument sbd;
    private final Resource asic;

    public SvarInnPackage(StandardBusinessDocument sbd, Resource asic) {
        this.sbd = sbd;
        this.asic = asic;
    }

    public StandardBusinessDocument getSbd() {
        return sbd;
    }

    public Resource getAsic() {
        return asic;
    }

}

File: integrasjonspunkt/src/main/kotlin/no/difi/meldingsutveksling/fiks/svarinn/SvarInnPackage.kt
package no.difi.meldingsutveksling.fiks.svarinn

import no.difi.meldingsutveksling.domain.sbdh.StandardBusinessDocument
import org.springframework.core.io.Resource

data class SvarInnPackage(val sbd: StandardBusinessDocument, val asic: Resource)